lib/ws_servers/api/algos/algo_worker.js

```diff
@@ -165,6 +165,20 @@ class AlgoWorker {
       })
   }
 
+  async closeActiveAlgosOnHost () {
+    const promises = []
+    this.host.getAOInstances()
+      .forEach((aoInstance) => {
+        const { state = {} } = aoInstance
+        const { gid, active } = state
+        if (!active) {
+          return
+        }
+        promises.push(this.cancelOrder(gid))
+      })
+    await Promise.all(promises)
```

Comment:
should we use allSettled in this? and cancel the biggest amount if an error occurs?